modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableColumnHeader.java

```diff
@@ -660,7 +663,7 @@ protected void resizeColumnToFitContent(int maxRows) {
             cell.updateIndex(row);
 
             if ((cell.getText() != null && !cell.getText().isEmpty()) || cell.getGraphic() != null) {
-                tableSkin.getChildren().add(cell);
+                tableRow.applyCss();
                 cell.applyCss();
```

Comment:
Just wondering: Is `cell.applyCss();` still needed here?

Reply:
Right, this is no longer needed. I'll remove it.